serialization/ascii/marshal.go

```diff
@@ -0,0 +1,28 @@
+package ascii
+
+import (
+	"reflect"
+)
+
+func Marshal(value interface{}) ([]byte, error) {
+	switch v := value.(type) {
```

Comment:
I see that you missing handler for `unsetColumn` and it seems wrong to me, am I wrong ?

Reply:
PR #343 was created.